src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -1,107 +1,107 @@
-	}
-
-	@Then("^the download button title should read as 'Download SED Details'$")
-	public void the_download_button_title_should_read_as_Download_SED_Details() throws Throwable {
-		
-	assertTrue(driver.getPageSource().contains("Download SED Details"));
-	}
-	
-	@Then("^the criteria number should also include the criteria title$")
-	public void the_criteria_number_should_also_include_the_criteria_title() throws Throwable {
-		assertTrue(driver.getPageSource().contains("170.315 (a)(1): Computerized Provider Order Entry (CPOE) - Medications"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(2): CPOE - Laboratory"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(3): CPOE - Diagnostic Imaging"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(4): Drug-Drug, Drug-Allergy Interaction Checks for CPOE"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(5): Demographics"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(6): Problem List"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(8): Medication Allergy List"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(14): Implantable Device List"));
-		assertTrue(driver.getPageSource().contains("170.315 (b)(3): Electronic Prescribing"));
-		assertTrue(driver.getPageSource().contains("170.315 (a)(7): Medication List"));
-	}
-		
-	
-	@Given("^I am on the details page of listing \"([^\"]*)\"$")
-	public void i_open_SED_accordion(String arg3) throws Throwable {
-		driver.get("https://chpl.ahrqstg.org/#/product/" + arg3);
-		listingDetails_page.sed_details_accordion_open(driver);
-	}
-	
-	@Then("^there should be text 'No Certification Criteria were tested for SED'$")
-	public void there_should_be_text_No_Certification_Criteria_were_tested_for_SED() throws Throwable {
-			
-		assertTrue(driver.getPageSource().contains("No Certification Criteria were tested for SED."));
-	}
-
-}
-
+package stepDefinitions;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import java.util.List;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import pageObjects.listingDetails_page;
+
+import static org.junit.Assert.assertTrue;
+
+public class SEDTweaksSteps {
+
+	public WebDriver driver;
+	
+	public SEDTweaksSteps()
+	{
+		driver = hooks.driver;
+		
+	}
+
+ 	
+	@Given("^I am on the Details page of Listing \"([^\"]*)\"$")
+	public void i_am_on_a_listing_s_details_pages(String arg1) throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/product/" + arg1);
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		listingDetails_page.load_listings_details_page(driver); 	    
+	}
+
+	
+	@Then("^I should see the accordions in the following order: Criteria, CQMs, SED, G1/G2, Surveillance, Additional Info$")

```

Comment:
This test doesn't test what it says it tests. It says it's testing ordering, but it's actually testing existence of. I think the reason for the original test was to test ordering, so the logic below should probably change. It wouldn't be wrong to _also_ have a test that tests the existence of, especially as different listings will have different sets of "existing" panels